cumulus/ansible/tasks/playbooks/roles/ec2-pod/tasks/launch.yml

```diff
@@ -0,0 +1,149 @@
+---
+# Note that we don't include master/node ami variables - these
+# could be conditionally checked, but because the next thing we
+# do is try to find the master/node ami if they aren't defined
+# it seems unnecessary.
+- name: Fail early if required variables are not defined
+  fail:
+    msg: "Variable {{ item }} is not defined"
+  when: item not in hostvars[inventory_hostname]
+  with_items:
+    - aws_keyname
+    - master_instance_type
+    - node_instance_type
+    - node_instance_count
+    - cluster_id
+    - cluster_region
+    - cluster_zone
+
+
+- name: Define default firewall rules if not provided
+  set_fact:
+    ec2_pod_rules:
+      - proto: tcp # ssh
+        from_port: 22
+        to_port: 22
+        cidr_ip: "{{ ssh_cidr_ip }}"
+
+      - proto: all
+        group_name: "ec2_pod_{{cluster_id}}"
+        group_desc: "Ec2 security group for cluster {{ cluster_id }}"
+  when: ec2_pod_rules is not defined
+
```

Comment:
After talking with @kotfic in person. We don't have to pass in ssh_cidr_ip, we can pass in an actual address for example 0.0.0.0/0 to open ssh to everyone. In the case of group_name we should allow this to be configured so that it can be used for this sort of configuration, rather than generate it. Otherwise its a little confusing for an application to have to use a group name for configuration which they didn't provide.